tests/test-json-comments-controller.php

```diff
@@ -116,6 +117,13 @@ public function test_get_comment_invalid_id() {
 		$this->assertErrorResponse( 'json_comment_invalid_id', $response, 404 );
 	}
 
+	public function test_get_comment_invalid_context() {
+		$request = new WP_JSON_Request( 'GET', sprintf( '/wp/comments/%s', $this->approved_id ) );
+		$request->set_param( 'context', 'edit' );
+		$response = $this->server->dispatch( $request );
+		$this->assertErrorResponse( 'json_comment_cannot_view', $response, 403 );
```

Comment:
With the latest merge of permissions checks, I'd expect this to return a generic `json_forbidden` now instead.